### PR TITLE
added traddify  folder which contains computed blockchain data tvl to the projects

### PR DESCRIPTION
**Pull request by Traddify**
**NOTE**

> If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).

1. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
2. The protocol is usually listed within 24 hours of merging the PR
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/traddify exports TVL: 
        
 
 
    kava                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/traddify exports TVL: 
        
 
 
    kava                      0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/traddify exports TVL: 
        
 
 
    kava                      0
    
    total                    0

**Comment by g1nt0ki** [comment]
> **NOTE**
> 
> > If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> 
>     1. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
> 
>     2. The protocol is usually listed within 24 hours of merging the PR
> 
>     3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
> 
>     4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
> 
>     5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
> 
>     6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap
> 
> 
> ##### Name (to be shown on DefiLlama):
> ##### Twitter Link:
> ##### List of audit links if any:
> ##### Website Link:
> ##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
> ##### Current TVL:
> ##### Chain:
> ##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
> ##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
> ##### Short Description (to be shown on DefiLlama):
> ##### Token address and ticker if any:
> ##### Category (full list at https://defillama.com/categories) *Please choose only one:
> ##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
> ##### forkedFrom (Does your project originate from another project):
> ##### methodology (what is being counted as tvl, how is tvl being calculated):

can you fill the project details?

File: projects/traddify/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { sumTokens2, nullAddress } = require('../helper/unwrapLPs')
+
+const contract = "0xA7f3d2dEa7a53E7A9FEbBdE5Cf7C69d39D065030"
+
+async function tvl(time, ethBlock, { kava: block }) {
+  return sumTokens2({ tokens: [nullAddress], owner: contract, block, chain: 'kava', })
```
Comment (g1nt0ki) [inline projects/traddify/index.js:6]: change nullAddress to `"0xc86c7C0eFbd6A49B35E8714C5f59D99De09A225b" ` if you want to include kava

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by llamatester** [comment]
The adapter at projects/traddify exports TVL: 
        
 
 
    kava                      19.358999999999998
    
    total                    19.358999999999998

**Pull request merged**